### PR TITLE
Docs: Add videos to What's new 11.2

### PR DESCRIPTION
**Pull request by imatwawana**
To be merged when the main Grafana 11.2 youtube video is listed publicly.

### CODE REVIEW

**Review by jdbaldry — APPROVED** [review]
LGTM

File: docs/sources/whatsnew/whats-new-in-v11-2.md [hunk 1]
```diff
@@ -152,6 +158,8 @@ To try out the new alert history page, enable the `alertingCentralAlertHistory`
 
 {{< figure src="/media/docs/alerting/alert-state-history.png" alt="Alert state history page" >}}
 
+{{< youtube id="0fNtby8ieEw" >}}
```
Comment (jdbaldry) [inline docs/sources/whatsnew/whats-new-in-v11-2.md:161]: This looks to still be in draft

File: docs/sources/whatsnew/whats-new-in-v11-2.md [hunk 2]
```diff
@@ -110,7 +112,7 @@ Data links in canvas elements can also be configured to open with a single click
 
 As part of this improvement, we've also added the ability to control the order in which data links are displayed by dragging and dropping them. This improvement has been added to all visualizations.
 
-{{< video-embed src="/media/docs/grafana/panels-visualizations/canvas-oneclick-tooltips-v4-11.2.mp4" >}}
+{{< youtube id="zOsM8VqwYpw" >}}
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v11-2.md:115]:
```suggestion
{{< video-embed src="/media/docs/grafana/panels-visualizations/canvas-oneclick-tooltips-v4-11.2.mp4" >}}

<!-- {{< youtube id="zOsM8VqwYpw" >}} -->
```

File: docs/sources/whatsnew/whats-new-in-v11-2.md [hunk 3]
```diff
@@ -38,6 +38,8 @@ This intuitive UI offers real-time updates on your migration status, making your
 
 Ready to make the move? Explore our [migration guide](https://grafana.com/docs/grafana-cloud/account-management/migration-guide/) to learn more about the Cloud Migration Assistant today and begin your effortless transition to Grafana Cloud.
 
+{{< youtube id="66W1UMHtX3U" >}}
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v11-2.md:41]:
```suggestion
<!-- {{< youtube id="66W1UMHtX3U" >}} -->
```

**Pull request merged**